net-utils/src/lib.rs

```diff
@@ -36,11 +36,6 @@ pub type PortRange = (u16, u16);
 pub const VALIDATOR_PORT_RANGE: PortRange = (8000, 10_000);
 pub const MINIMUM_VALIDATOR_PORT_RANGE_WIDTH: u16 = 17; // VALIDATOR_PORT_RANGE must be at least this wide
 
-#[cfg(not(any(windows, target_os = "ios")))]
-const DEFAULT_RECV_BUFFER_SIZE: usize = 64 * 1024 * 1024; // 64 MB - Doubled to 128MB by the kernel
-#[cfg(not(any(windows, target_os = "ios")))]
-const DEFAULT_SEND_BUFFER_SIZE: usize = 64 * 1024 * 1024; // 64 MB - Doubled to 128MB by the kernel
```

Comment:
I think this is line with intent from https://github.com/anza-xyz/agave/pull/3929#discussion_r1882768052, but just calling out this a potential change in behavior. I'm fine with it tho as:
- People who aren't knowledgable on this probably copy/pasted tuning guide
- People who are knowledgable probably set custom values
- Subsequent PR(s) will set intelligent values, and those will hopefully come in quicker with the groundwork laid in this PR

Reply:
i had added these lines in a recent/previous PR: https://github.com/anza-xyz/agave/pull/3594. But that same PR also deleted the tuning guide line to set socket buffer sizes via sysctl: https://github.com/anza-xyz/agave/pull/3594/files#diff-74b0da1ce50f08d2801b910a1f7f71fc6081fa7ea0cf093a255b390956552ee7.